lib/main.dart

```diff
@@ -14,12 +15,10 @@ class MyApp extends StatelessWidget {
       debugShowCheckedModeBanner: false,
       title: 'Tasks',
       getPages: AppPages.pages,
-      
```

Comment:
Why initial route is login where is splash screen route ? and please do not make regular changes in main.dart